graphql_compiler/query_formatting/common.py

```diff
@@ -1,22 +1,109 @@
 # Copyright 2017-present Kensho Technologies, LLC.
 """Safely insert runtime arguments into compiled GraphQL queries."""
+import datetime
+import decimal
+
+import arrow
+from graphql import GraphQLBoolean, GraphQLFloat, GraphQLID, GraphQLInt, GraphQLList, GraphQLString
 import six
 
 from ..compiler import GREMLIN_LANGUAGE, MATCH_LANGUAGE, SQL_LANGUAGE
+from ..compiler.helpers import strip_non_null_from_type
 from ..exceptions import GraphQLInvalidArgumentError
+from ..schema import GraphQLDate, GraphQLDateTime, GraphQLDecimal
 from .gremlin_formatting import insert_arguments_into_gremlin_query
 from .match_formatting import insert_arguments_into_match_query
 from .sql_formatting import insert_arguments_into_sql_query
 
 
+def _check_is_string_value(value):
+    """Raise if the value is not a proper utf-8 string."""
+    if not isinstance(value, six.string_types):
+        if isinstance(value, bytes):  # should only happen in py3
```

Comment:
Bytes are not strings, I think this should be an error. `GraphQLString` should only accept `six.string_types`.